web/dispers/dispers.go

```diff
@@ -16,18 +17,31 @@ CONCEPT INDEXOR'S ROUTES : those functions are used on route ./dispers/conceptin
 
 func createConcept(c echo.Context) error {
 
-	// TODO: Get concept from body
-
-	// TODO: Decrypt concepts
-
-	// TODO: Create array of hashes
-
-	// TODO: Call HashMeThat for each concept
-	hash, err := enclave.HashMeThat(concept)
+	// Get concept from body
+	var in dispers.InputCI
+	if err := json.NewDecoder(c.Request().Body).Decode(&in); err != nil {
+		return c.JSON(http.StatusOK, echo.Map{
+			"ok":    false,
+			"Error": err,
+		})
+	}
```

Comment:
The errors shouldn't be sent with the HTTP "ok" status. See the differents status [here](https://golang.org/pkg/net/http/) and how the errors are handled in the stack.
Also, there is no need to send `"ok": false` along the error, it is already implied by the error code.